### DISCUSSION TITLE
Welcome to Fork CMS!

### DISCUSSION THREAD

**Discussion by jessedobbelaere** [forkcms/forkcms#3252] [General]
![fork_features](https://user-images.githubusercontent.com/1352979/101530253-f47ff600-3991-11eb-8ecb-e2fef5eac6c0.png)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * [❓  Ask questions](https://github.com/forkcms/forkcms/discussions?discussions_q=category%3AHelp) you’re wondering about. We also have a [![Slack channel](https://imgur.com/zXuvRdw.png) Slack Channel](https://fork-cms.herokuapp.com)
  * [💡  Share ideas](https://github.com/forkcms/forkcms/discussions?discussions_q=category%3A%22Ideas+%2F+Feature+Requests%22).
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by carlo-ztl** [comment]
The link to the Slack channel is currently broken, it points to a non-existing channel on herokuapp. The same broken link is present everywhere (official web site too).
